akka-stream/src/main/mima-filters/2.5.20.backwards.excludes

```diff
@@ -10,3 +10,6 @@ ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.io.OutputStreamSou
 ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.StreamSupervisor$PrintDebugDump$")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.fusing.GraphInterpreter.dumpWaits")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.fusing.GraphInterpreterShell.dumpWaits")
+
+# #26246 Add asJava/asScala to RunnableGraph, not to be extended by users
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.javadsl.RunnableGraph.asScala")
```

Comment:
how could it have passed mima checks previously? or is it not needed?

Reply:
I was thinking that it got pulled in anyway, but I didn't dig into how the filename matching works.